tools/hooks/requirements_changed.rb

```diff
@@ -0,0 +1,34 @@
+REPO_DIR = File.expand_path('../../../', __FILE__)
+
+REQUIREMENTS = {
+  "requirements.txt" => "pip install -r requirements.txt",
```

Comment:
oh, that is correct; this is a version of a script I've been using for personal projects forever, including python ones.

I don't think we're using bower, either.

I have no opinion on keeping in (currently) unused patterns or not; I'd be happy to remove it if you do.